src/main/java/org/folio/inventory/common/MessagingContext.java

```diff
@@ -24,6 +24,11 @@ public String getOkapiLocation() {
     return getHeader("okapiLocation");
   }
 
+  @Override
+  public String getUserId() {
+    return getHeader("userId");
```

Comment:
There appears to be no mapping to this header when messages are created (see `IngestMessages`). Meaning this value will always be null.

If this is deliberate, I think it could be easier to follow by returning null explicitly.

Does this work because the MODS Ingestion does not create any circulation notes?

Reply:
Yes, MODS Ingestion does not create any circulation notes. Changed to return null explicitly.